rosidl_typesupport_cpp/test/test_message_type_support_dispatch.cpp

```diff
@@ -124,3 +125,24 @@ TEST(TestMessageTypeSupportDispatch, get_handle_function) {
       &type_support_cpp_identifier,
       "test_type_support4"), nullptr);
 }
+
+TEST(TestMessageTypeSupportDispatch, maybe_fail_test)
```

Comment:
Can you add to the test name the method under tests? probably something like `get_message_typesupport_maybe_fail_test`

Reply:
Agreed. Fixed here and in service test.